homeassistant/components/lacrosse_view/sensor.py

```diff
@@ -52,6 +58,46 @@ class LaCrosseSensorEntityDescription(
     "kilometers_per_hour": "km/h",
     "inches": "in",
 }
+SENSOR_DESCRIPTIONS = {
+    "Temperature": LaCrosseSensorEntityDescription(
+        key="",
```

Comment:
Set the identifying field as key.
```suggestion
        key="Temperature",
```